xbmc/rendering/dx/DeviceResources.cpp

```diff
@@ -195,13 +195,14 @@ bool DX::DeviceResources::SetFullScreen(bool fullscreen, RESOLUTION_INFO& res)
 
   critical_section::scoped_lock lock(m_criticalSection);
 
-  CLog::LogF(LOGDEBUG, "switching to/from fullscreen (%f x %f)", m_outputSize.Width,
-             m_outputSize.Height);
-
   BOOL bFullScreen;
-  bool recreate = m_stereoEnabled != (CServiceBroker::GetWinSystem()->GetGfxContext().GetStereoMode() == RENDER_STEREO_MODE_HARDWAREBASED);
-
   m_swapChain->GetFullscreenState(&bFullScreen, nullptr);
+
+  CLog::LogF(LOGDEBUG, "switching from %s (%.0f x %.0f) to %s (%d x %d)",
```

Comment:
looking at the lines bellow, it looks like you wanted to avoid double spaces in this log line
```suggestion
  CLog::LogF(LOGDEBUG, "switching from %s(%.0f x %.0f) to %s(%d x %d)",
```

Reply:
you are right. thanks